### ISSUE TITLE
How to properly test/compile application and sync with git in this repo?

### ISSUE THREAD

**Issue by vanowm** [OrangeDrangon/android-messages-desktop#221]
It seems this repo is missing some scripts from original repo, hens can't use the same commands to test and compile actual application as per instructions in [original readme](https://github.com/chrisknepper/android-messages-desktop#user-content-development)

`npm start` is as far as I'm able to go with this.

Also, after `npm install` it adds a bunch of new files into the directory of the repo, how do we handle them to sync any changes we do to the code via git?

(Sorry, total newbie here)

Any suggestions? Thanks.

**Comment by OrangeDrangon** [comment]
The code is pretty ugly but feel free to take a look.

Steps to contribute:
Fork
Clone
Edit
`yarn install` for dependencies
`yarn start` to build and run assuming you have electron installed.

**Comment by vanowm** [comment]
Thanks.
What about creating executable: installer and/or portable versions?
I've tried `yarn dist` but it opens electron_builder.js file in my text editor.  (electron is installed with `npm install electron --save-dev`

**Comment by vanowm** [comment]
I've followed instructions from [here](https://www.npmjs.com/package/electron-builder#installation) and installed electron-builder (`yarn add electron-builder --dev`)

That didn't help.
Than, I've tried run `electron-builder --config electron-builder.js` in cmd, but it simply opens electron_builder.js in my editor. When I attempted run the same command in PowerShell, got error saying electron-builder is not a command.

However when I run `.\node_modules\.bin\electron-builder --config electron-builder.js` it worked!
After changing package.json line:
`    "dist": "yarn build && ./node_modules/.bin/electron-builder --config electron-builder.js",`

`yarn dist` now works too.
